bats_test/downloads/file-downloads/public_files.bats

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bats
+
+# Support lib
+load '/home/node/packages/node_modules/bats-support/load.bash'
+# File methods. DEPENDS on SUPPORT lib.
+load '/home/node/packages/node_modules/bats-file/load.bash'
+# Assertions.  DEPENDS on SUPPORT lib.
+load '/home/node/packages/node_modules/bats-assert/load.bash'
+
+DIR="$(cd "$(dirname "$BATS_TEST_FILENAME")" >/dev/null 2>&1 && pwd)"
+
+setup_file() {
+    run $DIR/pre_download.sh
+    echo "##### Downloaded 7 files" >>/home/node/bats.log
+}
+
+@test 'Verifies "1 space test.zip" file exists' {
+    assert_exist "$(pwd)/1 space test.zip"
```

Comment:
Shouldn't we use the `DIR` variable here?
```suggestion
    assert_exist "${DIR}/1 space test.zip"
```

Reply:
Noup. Stuff is downloaded in an external script whereas test asserts relative to where it is located.